src/Adapter/Alias/Repository/AliasRepository.php

```diff
@@ -106,6 +111,20 @@ public function get(AliasId $aliasId): Alias
         return $alias;
     }
 
+    public function aliasExists(string $alias, string $searchTerm): bool
+    {
+        $qb = $this->connection->createQueryBuilder()
+            ->addSelect('*')
```

Comment:
selecting only id should be faster in theory, and you don't need other columns
```suggestion
            ->addSelect('a.id_alias')
```